lmdeploy/turbomind/chat.py

```diff
@@ -20,23 +22,26 @@ def input_prompt(model_name):
     return '\n'.join(iter(input, sentinel))
 
 
-def valid_str(string, coding='utf-8'):
-    """decode text according to its encoding type."""
-    invalid_chars = [b'\xef\xbf\xbd']
-    bstr = bytes(string, coding)
-    for invalid_char in invalid_chars:
-        bstr = bstr.replace(invalid_char, b'')
-    ret = bstr.decode(encoding=coding, errors='ignore')
-    return ret
-
-
 def main(model_path: str,
          model_name: str = None,
          session_id: int = 1,
+         top_k: float = 40,
+         top_p: float = 0.8,
+         temperature: float = 0.8,
+         repetition_penalty: float = 1.0,
          cap: str = 'chat',
          tp: int = 1,
+         max_batch_size: int = 1,
+         model_format: str = None,
+         quant_policy: int = 0,
+         cache_max_entry_count: float = 0.8,
+         cache_block_seq_len: int = 64,
+         rope_scaling_factor: float = 0.0,
+         enable_prefix_caching: bool = False,
+         session_len: int = None,
          stream_output: bool = True,
          request_output_len: int = 1024,
+         chat_template: str = None,
```

Comment:
Do we need both `chat_template` and `chat_template_cfg`?

Reply:
`lmdeploy chat /path/to/model` will pass  `chat_template_config` 

`lmdeploy chat turbomind /path/to/model` may pass `chat_template`

Reply:
Are they the same? If so, we can change one of them